gst/nnstreamer/tensor_filter/tensor_filter_common.h

```diff
@@ -107,6 +107,7 @@ enum
   PROP_SHARED_TENSOR_FILTER_KEY,
   PROP_LATENCY_REPORT,
   PROP_INVOKE_DYNAMIC,
+  PROP_INVOKE_ASYNC,
```

Comment:
Don't add it in the middle. You are breaking ABI.

Reply:
Okay, I will update.

Reply:
I've updated it. Thank you.